Realm/RLMCollection.h

```diff
@@ -154,40 +172,63 @@ typedef RLM_CLOSED_ENUM(int32_t, RLMPropertyType);
 /**
  Returns all objects matching the given predicate in the collection.
 
- @param predicate   The predicate with which to filter the objects.
+ This is only supported for managed collections.
```

Comment:
Why is this on the base protocol then? We have managed types.

Reply:
This is just documenting existing behavior. The existence of the managed subclasses isn't part of the API, and the base class exposes all of the functions, including the ones which will just throw an exception in the unmanaged case.